kafkacl/charmcraft.yaml

```
@@ -0,0 +1,17 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+# This file is here only to ensure libraries can be fetched and updated.
+
+name: dummy-charm
+summary: Placeholder charm for lib refresh.
+description: Placeholder charm for lib refresh.
+
+type: charm
+bases:
+  - build-on:
+      - name: "ubuntu"
+        channel: "22.04"
+    run-on:
+      - name: "ubuntu"
+        channel: "22.04"
```

Switch to new syntax

```suggestion
type: charm
platforms:
  ubuntu@22.04:amd64:
```